src/features/withdraw/frames/steps/confirm-step.tsx

```
@@ -0,0 +1,72 @@
+              disabled={!consents}
+              loading={isLoading}
+            >
+              {t('withdraw.confirm.action')}
+            </Button>
+          </div>
+        }
+      >
+        <div className="flex flex-col gap-3">
+          {/* TODO: add loading and error or empty status */}
+          {consents?.map((consent) => (
+            <div className="flex items-center gap-3 rounded-lg border border-neutral-100 p-3">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)